pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "scikit-lego"
-version = "0.8.1"
```

Comment:
Technically ... we're not introducing breaking changes here, right? So we might also consider 0.8.2? Or am I missing something?

Reply:
True, let me change it in both places